app/ui-react/syndesis/src/modules/integrations/IntegrationEditorApp.tsx

```diff
@@ -29,7 +29,9 @@ const addStepPage = (
         ...s,
       })
     }
-    apiProviderHref={resolvers.integration.edit.editStep.apiProvider.review}
+    apiProviderHref={
+      resolvers.integration.edit.editStep.apiProvider.selectMethod
```

Comment:
I guess in this case this is kinda no-op, or should this point to what would be the operations page...  hrm...

Reply:
Yeah I put this in place for now so that nothing breaks, but it's easy to change to whatever you or @riccardo-forina suggest.